src/scripts/pagecontent.js

```diff
@@ -470,6 +470,7 @@ class PageContent extends H5P.EventDispatcher {
       this.container.classList.remove('h5p-interactive-book-navigation-hidden');
       setTimeout(function () {
         self.container.classList.add('h5p-interactive-book-navigation-open');
+        self.parent.mainWrapper.get(0).classList.add('h5p-interactive-book-navigation-open');
```

Comment:
Need a class on the parent to tell if the navigation is open/closed. I probably could have used a different classname but I felt the class name was descriptive for the purpose. Feel free to comment if you feel this should be changed.